package.json

```diff
@@ -85,6 +85,7 @@
     "classnames": "^2.2.5",
     "material-components-web": "^0.32.0",
     "material-radial-progress": "git+https://gist.github.com/02134901c77c5309924bfcf8b4435ebe.git",
+    "mozjpeg": "^6.0.0",
```

Comment:
Why is this needed? And if it is, would it make more sense to have it in the `codecs/mozjpeg_enc/package.json`?

Reply:
Gotcha. I’ll remove it.

Reply:
... or you will :D